lib/propmangle.js

```diff
@@ -47,25 +47,28 @@ function find_builtins() {
 
     // Compatibility fix for some standard defined globals not defined on every js environment
     var new_globals = ["Symbol", "Map", "Promise", "Proxy", "Reflect", "Set", "WeakMap", "WeakSet"];
+    var objects = {};
 
     for (var i = 0; i < new_globals.length; i++) {
         if (!global[new_globals[i]]) {
```

Comment:
This can be shorted to `objects[new_globals[i]] = global[new_globals[i]] || new Function();` - but even better:
```js
new_globals.forEach(function (new_global) {
    objects[new_global] = global[new_global] || new Function();
});
```